templates/install-wp-tests.sh

```diff
@@ -14,6 +14,12 @@ WP_VERSION=${5-latest}
 WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
 WP_CORE_DIR=${WP_CORE_DIR-/tmp/wordpress/}
 
+if [[ $WP_VERSION =~ [0-9]+\.[0-9]+(\.[0-9]+)? ]]; then
+	WP_TESTS_TAG="tags/$WP_VERSION"
+else
+	WP_TESTS_TAG='trunk'
```

Comment:
Hm. The challenge is that `WP_VERSION=latest` isn't the same as trunk. Not sure what we can do to accommodate.